labs/src/main/scala/org/scalalabs/basic/lab01/OOExercise.scala

```diff
@@ -40,6 +40,33 @@ import scala.language.implicitConversions
  *   of type [[org.scalalabs.basic.lab01.CurrencyConverter]]
  * - Use the implicit CurrencyConverter to do the conversion. 
  */
-class Euro {
+abstract class Currency(val symbol:String)
 
-}
\ No newline at end of file
+class Euro(val euro:Int, val cents:Int=0) extends Currency("EUR") with Ordered[Euro]{
+  def inCents = 100*euro + cents
+  def +(other:Euro):Euro = {
```

Comment:
The [Scala Style Guide](http://docs.scala-lang.org/style/types.html) recommends putting a space between the colon and the type name, like so:
```def +(other: Euro): Euro =```